backend/wait-for-db.py

```diff
@@ -0,0 +1,31 @@
+import os
+import asyncio
+import asyncpg
+import time
+
+database_url = os.environ.get("DATABASE_URL")
+database_url = "postgresql://user:password@db:5432/llama_app_db"
+max_attempts = 30
+sleep_interval = 1
```

Comment:
Thoughts on making this parameters for the `check_database_connection` method you've written below? 
e.g.
`async def check_database_connection(max_attempts: int = 30, sleep_interval: int = 1) -> None:`

Reply:
made necessary changes